tests/foreman/longrun/test_oscap.py

```diff
@@ -0,0 +1,222 @@
+from fauxfactory import gen_string
+from nailgun import entities
+from robottelo.api.utils import (
+    enable_rhrepo_and_fetchid,
+    promote,
+    upload_manifest,
+)
+from robottelo import manifests, ssh
+from robottelo.constants import (
+    ANY_CONTEXT,
+    DEFAULT_SUBSCRIPTION_NAME,
+    OSCAP_DEFAULT_CONTENT,
+    OSCAP_PERIOD,
+    OSCAP_PROFILE,
+    OSCAP_WEEKDAY,
+    PRDS,
+    REPOS,
+    REPOSET,
+)
+from robottelo.config import conf
+from robottelo.decorators import run_only_on
+from robottelo.test import UITestCase
+from robottelo.ui.factory import set_context, make_hostgroup, make_oscappolicy
+from robottelo.ui.session import Session
+from robottelo.vm import VirtualMachine
+
+
+@run_only_on('sat')
+class OpenScap(UITestCase):
+    """Implements Product tests in UI"""
+
+    @classmethod
+    def setUpClass(cls):
+        """
+
+        1. Create new organization and environment
+        2. Clone and upload manifest
+        3. Sync a RedHat repository
+        4. Create content-view
+        5. Add repository to contet-view
+        6. Promote/publish content-view
+        7. Create an activation-key
+        8. Add product to activation-key
+        """
+        cls.ak_name = gen_string('alpha')
+        org = entities.Organization().create()
+        cls.org_name = org.name
+        # The below repos is used for cv's.
+        repos = []
+        repo_values = [{'repo': REPOS['rhst6']['name'],
+                        'reposet': REPOSET['rhst6'],
+                        },
+                       {'repo': REPOS['rhst7']['name'],
+                        'reposet': REPOSET['rhst7'],
+                        }]
+        # step 1.2: Create new lifecycle environments
+        env = entities.LifecycleEnvironment(
+            organization=org
+        ).create()
+        cls.env_name = env.name
+        # step 2: Upload manifest
+        with open(manifests.clone(), 'rb') as manifest:
+            upload_manifest(org.id, manifest)
+        for value in repo_values:
+            # step 3.1: Enable RH repo and fetch repository_id
+            repository = entities.Repository(id=enable_rhrepo_and_fetchid(
+                basearch='x86_64',
+                org_id=org.id,
+                product=PRDS['rhel'],
+                repo=value['repo'],
+                reposet=value['reposet'],
+                releasever=None,
+            ))
+            # step 3.2: sync repository
+            repository.sync()
```

Comment:
It's just 10MB , so decided not to use. It's very fast from pune, on sesame it will be even more fast. :)